widgetssdk/src/main/res/values/strings.xml

```diff
@@ -56,8 +56,8 @@
     <string name="glia_dialog_engagement_ended_action_ok">@string/glia_dialog_ok</string>
 
     <!-- Overlay permissions required Dialog -->
-    <string name="glia_dialog_overlay_permissions_title">Overlay Permissions Required</string>
-    <string name="glia_dialog_overlay_permissions_message">The chat requires screen overlay permissions to show message bubbles (optional). Please enable them for your app.</string>
+    <string name="glia_dialog_overlay_permissions_title">Screen Overlay Permissions Required</string>
```

Comment:
everything should be capitalized here? I assume this is displayed all caps in the app, but nevertheless should be correct.